web/tests/composebox_typeahead.test.js

```diff
@@ -2,7 +2,6 @@
 
 const {strict: assert} = require("assert");
 
-const {mock_stream_header_colorblock} = require("./lib/compose");
```

Comment:
All calls to `mock_stream_header_colorblock` and the file it's in have been removed, right? So the implementation is now dead code, and should be removed too.

Reply:
The call to `mock_stream_header_colorblock` has not been completely removed. As of now, it is still being used in https://github.com/zulip/zulip/blob/main/web/tests/drafts.test.js#L5.